### ISSUE TITLE
Loss function names switch to loss values randomly

### ISSUE THREAD

**Issue by dolhasz** [comet-ml/issue-tracking#87]
I am exporting the name of my loss function as a hyperparameter. When the model is training, comet displays the loss funciton name correctly in the summary table. However the models which have finished training show the final loss in the same column (see image)

![losses](https://user-images.githubusercontent.com/13208660/43678778-85ae9746-9811-11e8-9d79-e6a0a2b87875.png)

This tens to happen randomly and dynamically (they sometimes all load fine and after a few seconds the trained ones will switch dyanmically to the numerical loss value).

**Comment by ceceshao1** [comment]
Thank you for reporting this @dolhasz! I'll ask the dev team to take a look today. Might reach out later to get access to this specific project.

**Comment by Nimrod007** [comment]
Hi @dolhasz - is this a metric and a parameter with the same name/key ? 
something like:
```python
    experiment.log_metric("key", 123)
    experiment.log_parameter("key", 'some other value')
```

**Comment by dolhasz** [comment]
That is likely - my parameter is called Loss and my metric too. 

I assumed that may be the case. Should I just change my parameter name?

**Comment by ceceshao1** [comment]
@dolhasz - yes, try naming your parameter something different. Let us know if that fixes the strange switching problem.

**Comment by dolhasz** [comment]
All works - sorry it took me a while to confirm! Issue solved.

**Comment by ceceshao1** [comment]
Thanks for confirming @dolhasz. Closing this issue now.
